great_expectations/checkpoint/v1_checkpoint.py

```diff
@@ -120,8 +129,97 @@ def run(
         batch_parameters: Dict[str, Any] | None = None,
         expectation_parameters: Dict[str, Any] | None = None,
     ) -> CheckpointResult:
-        raise NotImplementedError
+        run_id = RunIdentifier(run_time=dt.datetime.now(dt.timezone.utc))
+        run_results = self._run_validation_definitions(
+            batch_parameters=batch_parameters,
+            expectation_parameters=expectation_parameters,
+            run_id=run_id,
+        )
+        self._run_actions(run_results=run_results)
+
+        return CheckpointResult(
+            run_id=run_id,
+            run_results=run_results,
+            checkpoint_config=self,
+            validation_result_url=None,  # TODO: Need to plumb from actions
```

Comment:
`None` for now - will follow up shortly

Reply:
Is the validation result url only appropriate for cloud checkpoint runs? I would remove this attribute for now until we plumb it in.

Reply:
I believe it is but it is a top-level param (just being explicit about the default here). I'd vote to keep this as-is but let's discuss